service/routers/users.js

```diff
@@ -0,0 +1,55 @@
+import { Router } from 'express'
+import { User } from '../models/User.js'
+
+const usersRouter = Router()
+
+const getUsers = async (req, res) => {
+  try{
+    const {userId} = req.user
+    const usr = await User.findById(userId)
+  if(usr.role === 'admin' || usr.role === 'superuser'){
```

Comment:
What role? Where are you defining this?

Reply:
roles are defined in the User.js model. I will make sure they are shown.